src/webui/api/torrentscontroller.h

```diff
@@ -46,42 +46,42 @@ private slots:
     void filesAction();
     void pieceHashesAction();
     void pieceStatesAction();
-    void resumeAction();
-    void pauseAction();
-    void recheckAction();
-    void reannounceAction();
-    void renameAction();
-    void setCategoryAction();
-    void createCategoryAction();
-    void editCategoryAction();
-    void removeCategoriesAction();
+    void resumeAction() const;
+    void pauseAction() const;
+    void recheckAction() const;
+    void reannounceAction() const;
+    void renameAction() const;
+    void setCategoryAction() const;
+    void createCategoryAction() const;
+    void editCategoryAction() const;
+    void removeCategoriesAction() const;
     void categoriesAction();
```

Comment:
I don't understand why you mark some of them as `const` and skip others... Maybe you shouldn't have messed with this job?

Reply:
I only converted the methods that don't call `setResult()` because marking them as `const` will result in a compiler error.
As you can probably tell my experience in c++ is very limited. If you want I can remove the last commit and somebody else can take over.

Reply:
>As you can probably tell my experience in c++ is very limited. If you want I can remove the last commit and somebody else can take over.

Yes, I want. Seems it requires more deeper cleanups.

Reply:
I also removed const from markAsReadAction since it would look weird if only one function was const.